### PR TITLE
New package: xapian-omega-1.4.7_1

### PR DESCRIPTION
**Pull request by xelxebar**

### CODE REVIEW

File: srcpkgs/xapian-omega/template [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Template file for 'xapian-omega'
+pkgname=xapian-omega
+version=1.4.7
+revision=1
+build_style=gnu-configure
+hostmakedepends="perl"
+makedepends="file-devel libuuid-devel pcre-devel xapian-core-devel zlib-devel"
+short_desc="Document indexer based on Xapian"
+maintainer="B. Wilson <x@wilsonb.com>"
+license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/xapian-omega/template]: Please use SPDX identifier

File: srcpkgs/xapian-omega/template [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Template file for 'xapian-omega'
+pkgname=xapian-omega
+version=1.4.7
+revision=1
+build_style=gnu-configure
+hostmakedepends="perl"
+makedepends="file-devel libuuid-devel pcre-devel xapian-core-devel zlib-devel"
+short_desc="Document indexer based on Xapian"
+maintainer="B. Wilson <x@wilsonb.com>"
+license="GPL-2.0"
```
Comment (maxice8) [inline srcpkgs/xapian-omega/template]: SPDX identifier requires -or-later or -only specifier

Comment (xelxebar) [reply srcpkgs/xapian-omega/template]: Oof. Thanks.

**Comment by maxice8** [comment]
needs a conf_files entry for /etc/omega.conf

**Pull request closed without merging**